exercises/affine-cipher/description.md

```diff
@@ -41,9 +40,10 @@ The decryption function is:
 D(y) = (a^-1)(y - b) mod m
 ```
 
-- where `y` is the numeric value of an encrypted letter, ie. `y = E(x)`
-- it is important to note that `a^-1` is the modular multiplicative inverse (MMI)
-  of `a mod m`
+Where:
```

Comment:
I was initially uncertain that the "Where" should be lifted out of the list, because only the term `y` is being defined anew, and the other things are just additional notes, but after thinking about it I see that it does make sense if each individual item is prefixed with the word "Where", so that seems fine. Even though the only term being defined is `y`, the operation `a^-1` is being defined as well.